Signal/src/ViewControllers/ConversationView/Emoji Picker/EmojiPickerCollectionView.swift

```diff
@@ -174,11 +174,11 @@ class EmojiPickerCollectionView: UICollectionView {
         var newRecentEmoji = recentEmoji
 
         // Remove any existing entries for this emoji
-        newRecentEmoji.removeAll { emoji == $0 }
+        newRecentEmoji.removeAll { emoji.baseEmoji == $0.baseEmoji }
         // Insert the selected emoji at the start of the list
         newRecentEmoji.insert(emoji, at: 0)
         // Truncate the recent emoji list to a maximum of 50 stored
-        newRecentEmoji = Array(newRecentEmoji[0..<min(50, newRecentEmoji.count)])
+        newRecentEmoji = Array(newRecentEmoji.suffix(50))
```

Comment:
I tested locally without these lines, and things behaved the way I expected. I think we can skip these changes?